src/components/HomePage/RepairServices/data.js

```diff
@@ -0,0 +1,29 @@
+export const servicesData1 = [
```

Comment:
you may convert them to objects to know exactly what are working, not a most but better